README.md

```diff
@@ -121,6 +123,21 @@ You will be asked to supply the following information:
 	```
 	(On windows use `activate env` to activate the conda environment.)
 
+## Continuous integration with Github Actions
+
+The template has two Ci workflows. They can be found in **.github/workflows** folder.
+
+1. **build.yml**
+
+This workflow install the dependencies, builds the package and runs tests.
+
+2. **pypi_deploy.yml**
+
+This workflow pushes the package to [PYPI](https://pypi.org/). This action will require PYPI token to be stored as [Github secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).
```

Comment:
What should the secret be named like? PYPI_PASSWORD?

Reply:
done